app/components/display/alert_component/alert_component_controller.js

```diff
@@ -0,0 +1,7 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
```

Comment:
Add here an option to close the alert automatically after some time (for example 500ms)